File: CommonLibraries/Common/Scripts/Exceptions/FileEmptyException.cs
using System;
using System.IO;

namespace Uheaa.Common.Scripts.Exceptions
{
    class FileEmptyException : FileNotFoundException
    {
        public FileEmptyException()
            : base()
        {
        }

        public FileEmptyException(string message)
            : base(message)
        {
        }

        public FileEmptyException(string message, Exception innerException)
            : base(message, innerException)
        {
        }

        public FileEmptyException(string message, string fileName)
            : base(message, fileName)
        {
        }

        public FileEmptyException(string message, string fileName, Exception innerException)
            : base(message, fileName, innerException)
        {
        }
    }
}

File: N/NSFREVENTR/Q/Q/Common/FileEmptyException.vb
Public Class FileEmptyException
    Inherits System.IO.FileNotFoundException

    Public Sub New()
        MyBase.New()
    End Sub

    Public Sub New(ByVal message As String)
        MyBase.New(message)
    End Sub

    Public Sub New(ByVal message As String, ByVal innerException As Exception)
        MyBase.New(message, innerException)
    End Sub

    Public Sub New(ByVal message As String, ByVal fileName As String)
        MyBase.New(message, fileName)
    End Sub

    Public Sub New(ByVal message As String, ByVal fileName As String, ByVal innerException As Exception)
        MyBase.New(message, fileName, innerException)
    End Sub
End Class
